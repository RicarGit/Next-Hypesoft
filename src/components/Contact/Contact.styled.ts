import styled from "styled-components"

export const ContactSection = styled.section`
  width: 100%;
  margin-top: 231px;
  display: flex;
`

export const Container = styled.div`
  width: 1320px;
  height: 536px;
  margin: 0 auto;
  display: flex;
  gap: 24px;
`

export const ContactFormContainer = styled.div`
  width: 760px;
  height: 536px;
  display: flex;
  flex-direction: column;
  border-radius: 8px;
  padding: 48px;
  gap: 24px;
  background-color: ${({ theme }) => theme.colors.white};
`

export const ContactFormHeading = styled.h2`
  font-size: 32px;
  font-weight: 500;
  line-height: 40px;
  color: ${({ theme }) => theme.colors.primary};
`

export const ContactInfo = styled.div`
  position: relative;
  width: 536px;
  height: 536px;
  padding: 48px;
  display: flex;
  flex-direction: column;
  border-radius: 8px;
  gap: 26px;
  background-color: ${({ theme }) => theme.colors.white};
`

export const ContactInfoHeading = styled.h2`
  font-size: 32px;
  font-weight: 500;
  line-height: 40px;
  color: ${({ theme }) => theme.colors.primary};
`

export const AddressesContainer = styled.address`
  height: 376px;
  display: flex;
  flex-direction: column;
  justify-content: space-between;
`

export const SocialMediaContainer = styled.div`
  height: 60px;
  display: flex;
  gap: 12px;
`

export const pinkDotsImageWrapper = styled.div`
  position: absolute;
  top: -34px;
  right: -76px;
`