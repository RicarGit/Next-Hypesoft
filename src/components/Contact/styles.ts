import styled from "styled-components"

export const ContactSection = styled.section`
  width: 100%;
  height: 744px;
  position: absolute;
  display: flex;
  align-items: center;
  top: 5316px;
`

export const Container = styled.div`
  width: 1320px;
  height: 536px;
  margin: 0 auto;
  display: flex;
  gap: 24px;
`

export const ContactFormContainer = styled.div`
  width: 760px;
  height: 536px;
  display: flex;
  flex-direction: column;
  border-radius: 8px;
  padding: 48px;
  gap: 24px;
  background-color: ${({ theme }) => theme.colors.white};
`

export const ContactFormHeading = styled.h2`
  font-size: 32px;
  font-weight: 500;
  line-height: 40px;
  color: ${({ theme }) => theme.colors.primary};
`

export const ContactForm = styled.form`
  width: 664px;
  height: 288px;
  display: flex;
  flex-wrap: wrap;
  gap: 16px;
`

export const FormInput = styled.input`
  height: 52px;
  width: 324px;
  font-size: 16px;
  font-weight: 400;
  line-height: 24px;
  border-radius: 4px;
  padding: 24px;
  outline: none;
  color: ${({ theme }) => theme.colors.gray[500]};
  border: 1px solid ${({ theme }) => theme.colors.gray[100]};

  &.fullWidth {
    width: 100%;
  }
`

export const FormTextArea = styled.textarea`
  width: 100%;
  height: 152px;
  border-radius: 4px;
  padding: 24px;
  outline: none;
  resize: none;
  color: ${({ theme }) => theme.colors.gray[500]};
  border: 1px solid ${({ theme }) => theme.colors.gray[100]};
`

export const SubmitButton = styled.button`
  height: 56px;
  padding: 0 32px;
  margin-top: 16px;
  display: flex;
  align-items: center;
  font-size: 16px;
  font-weight: 600;
  line-height: 56px;
  letter-spacing: 0.02em;
  text-transform: uppercase;
  border: none;
  border-radius: 5px;
  gap: 14px;
  cursor: pointer;
  color: ${({ theme }) => theme.colors.white};
  background-color: ${({ theme }) => theme.colors.sec};
`

export const ContactInfo = styled.div`
  position: relative;
  width: 536px;
  height: 536px;
  padding: 48px;
  display: flex;
  flex-direction: column;
  border-radius: 8px;
  gap: 26px;
  background-color: ${({ theme }) => theme.colors.white};
`

export const ContactInfoHeading = styled.h2`
  font-size: 32px;
  font-weight: 500;
  line-height: 40px;
  color: ${({ theme }) => theme.colors.primary};
`

export const AddressesContainer = styled.address`
  height: 376px;
  display: flex;
  flex-direction: column;
  justify-content: space-between;
`

export const AddressInfoContainer = styled.div`
  height: 80px;
  display: flex;
  align-items: center;
  gap: 20px;
`

export const AddressIconWrapper = styled.div`
  width: 80px;
  height: 80px;
  border-radius: 50%;
  display: flex;
  justify-content: center;
  background-color: rgba(104, 79, 255, 0.1);
`

export const AddressContact = styled.p`
  font-size: 18px;
  font-weight: 400;
  font-style: normal;
  line-height: 28px;
  color: ${({ theme }) => theme.colors.primary};
`

export const SocialMediaContainer = styled.div`
  height: 60px;
  display: flex;
  gap: 12px;
`

export const SocialMediaButton = styled.div`
  width: 60px;
  height: 60px;
  padding: 20px;
  border-radius: 50%;
  cursor: pointer;
  border: 1px solid ${({ theme }) => theme.colors.gray[100]};
  background-color: ${({ theme }) => theme.colors.white};

  &.active {
    background-color: ${({ theme }) => theme.colors.sec};
    border: none;
  }
`

export const pinkDotsImageWrapper = styled.div`
  position: absolute;
  top: -34px;
  right: -76px;
`