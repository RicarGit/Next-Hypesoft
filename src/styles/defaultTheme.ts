import { DefaultTheme } from "styled-components"

const defaultTheme: DefaultTheme = {
  colors: {
    primary: '#242424',
    secondary: '#ED1CA6',
    sec: '#7D00FF',
    white: '#FFFFFF',
    gray: {
      100: '#DFE3EB',
      500: '#A5A5A5',
    },
  },
  font: {
    family: 'Inter',
    weigths: {
      regular: 400,
      medium: 500,
      semiBold: 600,
      bold: 700
    },
  },
}

export default defaultTheme