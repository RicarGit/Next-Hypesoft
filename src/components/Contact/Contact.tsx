import * as S from './Contact.styled'

import Image from 'next/image'
import { Assets } from '~/assets'

import {
  ContactSocialMediaButton,
  ContactForm,
  AddressInfo
} from '~/components'

export const Contact = () => {
  return (
    <S.ContactSection id='contact'>
      <S.ContactFormContainer>
        <S.ContactFormHeading>Let me know here.</S.ContactFormHeading>
        <ContactForm />
      </S.ContactFormContainer>

      <S.ContactInfo>
        <S.ContactInfoHeading>Get In Touch</S.ContactInfoHeading>
        <S.AddressesContainer>
          <AddressInfo
            wayToContact={Assets.ContactLocation}
            info1='House #5, Street Number #98,'
            info2='House #5, Street Number #98,'
            alt='location icon'
          />
          <AddressInfo
            wayToContact={Assets.ContactMail}
            info1='albert.design@gmail.com'
            info2='albert.flores@gmail.com'
            alt='mail icon'
          />
          <AddressInfo
            wayToContact={Assets.ContactPhone}
            info1='+55 955 258 2699'
            info2='+55 955 100 9449'
            alt='mail icon'
          />

          <S.SocialMediaContainer>
            <ContactSocialMediaButton
              logo={Assets.Dribble}
              alt='drible logo'
            />
            <ContactSocialMediaButton
              logo={Assets.Behance}
              alt='behance logo'
              className='active'
            />
            <ContactSocialMediaButton
              logo={Assets.Facebook}
              alt='facebook logo'
            />
            <ContactSocialMediaButton
              logo={Assets.Twitter}
              alt='twitter logo'
            />
            <ContactSocialMediaButton
              logo={Assets.Instagram}
              alt='instagram logo'
            />
            <ContactSocialMediaButton
              logo={Assets.YoutubeIcon}
              alt='youtube icon'
            />
          </S.SocialMediaContainer>
        </S.AddressesContainer>

        <S.pinkDotsImageWrapper>
          <Image
            src={Assets.PinkHorizontalDots}
            width={152}
            height={68}
            alt='pink dots'
          />
        </S.pinkDotsImageWrapper>
      </S.ContactInfo>
    </S.ContactSection>
  )
}