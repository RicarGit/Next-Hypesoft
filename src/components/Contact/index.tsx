import * as S from './styles'

import Image from 'next/image'
import sendIcon from './sendIcon.svg'
import contactLocation from './contactLocation.svg'
import contactMail from './contactMail.svg'
import contactPhone from './contactPhone.svg'
import facebook from 'components/shared/images/facebook.svg'
import behance from 'components/shared/images/behance.svg'
import dribble from 'components/shared/images/dribbble.svg'
import twitter from 'components/shared/images/twitter.svg'
import instagram from './instagram.svg'
import youtubeIcon from './youtubeIcon.png'
import horizontalPinkDots from './pink-horizontal-dots.svg'

const Contact = () => {
  return (
    <S.ContactSection id='contact'>
      <S.Container>

        <S.ContactFormContainer>
          <S.ContactFormHeading>Let me know here.</S.ContactFormHeading>

          <S.ContactForm method='POST'>
            <S.FormInput type='text' name='fullname' placeholder='Full name' required />
            <S.FormInput type='email' name='email' placeholder='Email address' required />
            <S.FormInput type='text' name='subjects' className='fullWidth' placeholder='Subjects' required />
            <S.FormTextArea placeholder='Message' />

            <S.SubmitButton type='submit'>
              Send Message
              <Image src={sendIcon} />
            </S.SubmitButton>

          </S.ContactForm>
        </S.ContactFormContainer>

        <S.ContactInfo>
          <S.ContactInfoHeading>Get In Touch</S.ContactInfoHeading>
          <S.AddressesContainer>
            <S.AddressInfoContainer>

              <S.AddressIconWrapper>
                <Image src={contactLocation} />
              </S.AddressIconWrapper>

              <S.AddressContact>
                House #5, Street Number #98,<br />
                brasilia-	70000-000, Brazil.
              </S.AddressContact>

            </S.AddressInfoContainer>
            <S.AddressInfoContainer>

              <S.AddressIconWrapper>
                <Image src={contactMail} />
              </S.AddressIconWrapper>

              <S.AddressContact>
                albert.design@gmail.com<br />
                albert.flores@gmail.com
              </S.AddressContact>

            </S.AddressInfoContainer>
            <S.AddressInfoContainer>

              <S.AddressIconWrapper>
                <Image src={contactPhone} />
              </S.AddressIconWrapper>

              <S.AddressContact>
                +55 955 258 2699<br />
                +55 955 100 9449
              </S.AddressContact>
            </S.AddressInfoContainer>

            <S.SocialMediaContainer>
              <S.SocialMediaButton>
                <Image src={dribble} />
              </S.SocialMediaButton>

              <S.SocialMediaButton className='active'>
                <Image src={behance} />
              </S.SocialMediaButton>

              <S.SocialMediaButton>
                <Image src={facebook} />
              </S.SocialMediaButton>

              <S.SocialMediaButton>
                <Image src={twitter} />
              </S.SocialMediaButton>

              <S.SocialMediaButton>
                <Image src={instagram} />
              </S.SocialMediaButton>

              <S.SocialMediaButton>
                <Image src={youtubeIcon} />
              </S.SocialMediaButton>
            </S.SocialMediaContainer>

          </S.AddressesContainer>

          <S.pinkDotsImageWrapper>
            <Image src={horizontalPinkDots} />
          </S.pinkDotsImageWrapper>
        </S.ContactInfo>

      </S.Container>
    </S.ContactSection>
  )
}

export default Contact