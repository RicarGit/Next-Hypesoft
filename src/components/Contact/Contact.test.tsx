import { render } from '@testing-library/react'
import { ThemeProvider } from 'styled-components'
import defaultTheme from 'styles/defaultTheme'
import Contact from '.'

describe('Contact', () => {
  it('should match snapshot', () => {
    const { container } = render(
      <ThemeProvider theme={defaultTheme}>
        <Contact />
      </ThemeProvider>
    )

    expect(container.firstChild).toMatchSnapshot()
  })
})