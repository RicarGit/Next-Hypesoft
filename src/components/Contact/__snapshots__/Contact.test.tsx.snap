// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact should match snapshot 1`] = `
<section
  class="sc-bczRLJ tUPnm"
  id="contact"
>
  <div
    class="sc-gsnTZi lbnUcY"
  >
    <div
      class="sc-dkzDqf eAKskj"
    >
      <h2
        class="sc-hKMtZM fnLlmU"
      >
        Let me know here.
      </h2>
      <form
        class="sc-jqUVSM dNgrcv"
        method="POST"
      >
        <input
          class="sc-kDDrLX kuCkTr"
          name="fullname"
          placeholder="Full name"
          required=""
          type="text"
        />
        <input
          class="sc-kDDrLX kuCkTr"
          name="email"
          placeholder="Email address"
          required=""
          type="email"
        />
        <input
          class="sc-kDDrLX kuCkTr fullWidth"
          name="subjects"
          placeholder="Subjects"
          required=""
          type="text"
        />
        <textarea
          class="sc-iqcoie jhRwXq"
          placeholder="Message"
        />
        <button
          class="sc-crXcEl jiBZvo"
          type="submit"
        >
          Send Message
          <span
            style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
          >
            <span
              style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
            >
              <img
                alt=""
                aria-hidden="true"
                src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2724%27%20height=%2724%27/%3e"
                style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
              />
            </span>
            <img
              alt="submit button icon"
              data-nimg="intrinsic"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
            />
            <noscript />
          </span>
        </button>
      </form>
    </div>
    <div
      class="sc-eCYdqJ FSAnU"
    >
      <h2
        class="sc-jSMfEi deuGrP"
      >
        Get In Touch
      </h2>
      <address
        class="sc-gKXOVf cnColw"
      >
        <div
          class="sc-evZas iTPdOz"
        >
          <div
            class="sc-breuTD cLOrYw"
          >
            <span
              style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
            >
              <span
                style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
              >
                <img
                  alt=""
                  aria-hidden="true"
                  src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2724%27%20height=%2724%27/%3e"
                  style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                />
              </span>
              <img
                alt="location icon"
                data-nimg="intrinsic"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
              />
              <noscript />
            </span>
          </div>
          <p
            class="sc-ksZaOG dDrInA"
          >
            House #5, Street Number #98,
            <br />
            House #5, Street Number #98,
          </p>
        </div>
        <div
          class="sc-evZas iTPdOz"
        >
          <div
            class="sc-breuTD cLOrYw"
          >
            <span
              style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
            >
              <span
                style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
              >
                <img
                  alt=""
                  aria-hidden="true"
                  src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2724%27%20height=%2724%27/%3e"
                  style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                />
              </span>
              <img
                alt="mail icon"
                data-nimg="intrinsic"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
              />
              <noscript />
            </span>
          </div>
          <p
            class="sc-ksZaOG dDrInA"
          >
            albert.design@gmail.com
            <br />
            albert.flores@gmail.com
          </p>
        </div>
        <div
          class="sc-evZas iTPdOz"
        >
          <div
            class="sc-breuTD cLOrYw"
          >
            <span
              style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
            >
              <span
                style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
              >
                <img
                  alt=""
                  aria-hidden="true"
                  src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2724%27%20height=%2724%27/%3e"
                  style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                />
              </span>
              <img
                alt="mail icon"
                data-nimg="intrinsic"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
              />
              <noscript />
            </span>
          </div>
          <p
            class="sc-ksZaOG dDrInA"
          >
            +55 955 258 2699
            <br />
            +55 955 100 9449
          </p>
        </div>
        <div
          class="sc-iBkjds fUNJbI"
        >
          <div
            class="sc-papXJ iUryHn"
          >
            <span
              style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
            >
              <span
                style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
              >
                <img
                  alt=""
                  aria-hidden="true"
                  src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2724%27%20height=%2724%27/%3e"
                  style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                />
              </span>
              <img
                alt="drible logo"
                data-nimg="intrinsic"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
              />
              <noscript />
            </span>
          </div>
          <div
            class="sc-papXJ iUryHn active"
          >
            <span
              style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
            >
              <span
                style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
              >
                <img
                  alt=""
                  aria-hidden="true"
                  src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2724%27%20height=%2724%27/%3e"
                  style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                />
              </span>
              <img
                alt="behance logo"
                data-nimg="intrinsic"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
              />
              <noscript />
            </span>
          </div>
          <div
            class="sc-papXJ iUryHn"
          >
            <span
              style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
            >
              <span
                style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
              >
                <img
                  alt=""
                  aria-hidden="true"
                  src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2724%27%20height=%2724%27/%3e"
                  style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                />
              </span>
              <img
                alt="facebook logo"
                data-nimg="intrinsic"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
              />
              <noscript />
            </span>
          </div>
          <div
            class="sc-papXJ iUryHn"
          >
            <span
              style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
            >
              <span
                style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
              >
                <img
                  alt=""
                  aria-hidden="true"
                  src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2724%27%20height=%2724%27/%3e"
                  style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                />
              </span>
              <img
                alt="twitter logo"
                data-nimg="intrinsic"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
              />
              <noscript />
            </span>
          </div>
          <div
            class="sc-papXJ iUryHn"
          >
            <span
              style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
            >
              <span
                style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
              >
                <img
                  alt=""
                  aria-hidden="true"
                  src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2724%27%20height=%2724%27/%3e"
                  style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                />
              </span>
              <img
                alt="instagram logo"
                data-nimg="intrinsic"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
              />
              <noscript />
            </span>
          </div>
          <div
            class="sc-papXJ iUryHn"
          >
            <span
              style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
            >
              <span
                style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
              >
                <img
                  alt=""
                  aria-hidden="true"
                  src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2724%27%20height=%2724%27/%3e"
                  style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                />
              </span>
              <img
                alt="youtube icon"
                data-nimg="intrinsic"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
              />
              <noscript />
            </span>
          </div>
        </div>
      </address>
      <div
        class="sc-ftvSup edxLOB"
      >
        <span
          style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
        >
          <span
            style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
          >
            <img
              alt=""
              aria-hidden="true"
              src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%2724%27%20height=%2724%27/%3e"
              style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
            />
          </span>
          <img
            alt="pink dots"
            data-nimg="intrinsic"
            decoding="async"
            src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
            style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
          />
          <noscript />
        </span>
      </div>
    </div>
  </div>
</section>
`;
